content/docs/reference/bearer-token-format.mdx

```diff
@@ -76,6 +84,14 @@ Set **Bearer Token Format** under **Proxy** settings in the Console:
 
 ![Set bearer token format in the Console](./img/global-settings/bearer-token-format.png)
 
+### Options
+
+- _Unset_: At the route level, use the global setting. At the global level, use "Default".
+- "Unknown": Same as "Default".
```

Comment:
Seems like we shouldn't have this value available in the UI, but I suppose this isn't the only place we have this, so let's leave it and document it for now and discuss how we can address these Unknowns where they exist.